### PR TITLE
Move authorization into the deployment engine

### PR DESCRIPTION
**Pull request by bear454**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ Engines are only loaded if the application is configured to do so. To load an en
 
 Since each engine may define UI elements in the workflow, the order engines are loaded determines the order of the menu entries in the application. The only exception is the 'Welcome' page, which is always first.
 
+### Deployment Engine
+
+One Engine in the workflow should be designated at as 'deployment engine'. This should define the overall workflow and the steps to be performed for the deployment, including authorization for the entire workflow. See [engines/rancher_on_eks/app/helpers/rancher_on_eks/authorization_helper.rb](engines/rancher_on_eks/app/helpers/rancher_on_eks/authorization_helper.rb) for an example.
```
[the remark this answers is not in the document]
Comment (bear454) [reply README.md:75]: Authentication happens at the application-level, using cloud-instance-credentials. This is authorization to move through the steps.

File: config/initializers/i18n_debug.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'i18n'
+
+module I18nLogging
+  def translate(locale, key, options = EMPTY_HASH)
+    puts "Translate: #{locale}: #{key}"
```
[the remark this answers is not in the document]
Comment (bear454) [reply config/initializers/i18n_debug.rb:5]: I wanted to log the translation keys. Yes, I'll switch it to the logger.

[the remark this answers is not in the document]
Comment (bear454) [reply config/initializers/i18n_debug.rb]: Resolved in ac44396

**Review by jesusbv — APPROVED** [review]
LGTM

**Pull request merged**